deepmd/infer/model_devi.py

```diff
@@ -127,8 +125,10 @@ def calc_model_devi(coord,
     >>> graphs = [DP("graph.000.pb"), DP("graph.001.pb")]
     >>> model_devi = calc_model_devi(coord, cell, atype, graphs)
     '''
-    if nopbc:
-        box = None
+    if not box:
```

Comment:
This will not work when `box` is `np.ndarray`.

```py
>>> import numpy as np
>>> not np.ones(3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

So `is not None` should be used instead.
```suggestion
    if box is not None:
```